src/actions/gifActions.js

```diff
@@ -0,0 +1,36 @@
+import { createAction } from 'redux-actions';
+
+const SEARCH_GIFS_FAILED = 'SEARCH_GIFS_FAILED';
+const SEARCH_GIFS_GET = 'SEARCH_GIFS_GET';
+const SEARCH_GIFS_RECEIVED = 'SEARCH_GIFS_RECEIVED';
+
+const failedSearchGifs = createAction(SEARCH_GIFS_FAILED);
+const receivedSearchGifs = (payload = {}) => ({ type: SEARCH_GIFS_RECEIVED, payload });
+const searchGifs = (payload = {}) => ({ type: SEARCH_GIFS_GET, payload });
```

Comment:
Why did you mix the usage of `createAction` and a manual declaration?

Reply:
I followed this files as examples: https://github.com/wizelineacademy/redux-course/blob/master/src/actions/cryptoPrices.js, https://github.com/wizelineacademy/redux-course/blob/master/src/actions/transactions.js. 
I didn't know that there's a different way to do it.